### PR TITLE
Adding temperatures

### PR DESCRIPTION
**Pull request by amp7368**
finished adding temperatures to the plugin. I had two commands that I turned into one using acf. Also made getDataFolder make a new folder for that module if it doesn't already exist

### CODE REVIEW

File: src/main/java/com/voltskiya/core/temperatures/TemperatureCommands.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+@CommandAlias("temperature")
+public class TemperatureCommands extends BaseCommand {
+    private File dataFolder;
+
+    public TemperatureCommands(File dataFolder) {
+        this.dataFolder = dataFolder;
+    }
+
+    @Subcommand("remove")
+    public boolean remove(@NotNull CommandSender commandSender, @Single String playerName) {
```
Comment (harrisonglp) [inline src/main/java/com/voltskiya/core/temperatures/TemperatureCommands.java]: You can just use the argument of Player to provide an online player, you should also annotate with CommandCompletion("@players") for completions.

File: src/main/java/com/voltskiya/core/temperatures/PlayerListener.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+public class PlayerListener implements Listener {
+    private final JavaPlugin plugin;
+
+    private final Map<UUID, WatchPlayer> watching = new HashMap<>();
+    private final Object watchingSyncObject = new Object();
```
Comment (harrisonglp) [inline src/main/java/com/voltskiya/core/temperatures/PlayerListener.java]: Should we just use a ConcurrentHashMap?

File: .gitignore [hunk 3]
```diff
@@ -1,22 +1,12 @@
...
+
+# Ignore Gradle build output directory
+build
+/build.gradle
+/gradlew
+/gradlew.bat
+/settings.gradle
+/libs/
+/gradle/
+/.idea/
```
Comment (harrisonglp) [inline .gitignore]: did you mean to delete the gitignore stuff I had?

File: src/main/java/com/voltskiya/core/temperatures/TemperatureCommands.java (hunk 1 above)
Comment (amp7368) [reply src/main/java/com/voltskiya/core/temperatures/TemperatureCommands.java]: That worked very well

File: src/main/java/com/voltskiya/core/temperatures/PlayerListener.java (hunk 2 above)
Comment (amp7368) [reply src/main/java/com/voltskiya/core/temperatures/PlayerListener.java]: Yes, we should and we will! :D

File: .gitignore (hunk 3 above)
Comment (amp7368) [reply .gitignore]: Nope, but that should be good now

**Review by harrisonglp — APPROVED** [review]

**Pull request merged**